.infrastructure/helm-chart/todoapp/templates/rbac.yml

```diff
@@ -0,0 +1,30 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: {{.Values.todoapp.RBAC.name}}
+  namespace: {{.Values.todoapp.namespace}}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: {{.Values.todoapp.namespace}}
+  name: {{.Values.todoapp.RBAC.name}}
```

Comment:
Ensure that the `todoapp.RBAC.name` and `todoapp.namespace` values are defined in your `values.yaml` file for the todoapp chart. If they are not defined, this could lead to issues when creating the Role. Consider providing default values or handling the case where they might be undefined.